### ISSUE TITLE
Safari: legend position will offset when zoom in

### ISSUE THREAD

**Issue by busyxiang** [apexcharts/apexcharts.js#2963]
### Description
Safari browser will offset legends when zoom in

### Steps to Reproduce

1. Open Safari browser
2. Open any demo charts with legends
3. Zoom in

### Expected Behavior
The legends' position should not offset when zoom in

### Actual Behavior
The legends' position offset when zoom in

### Screenshots
<img width="699" alt="Screenshot 2022-02-10 at 2 24 43 PM" src="https://user-images.githubusercontent.com/19426419/153350263-94a05ac2-e0e5-490a-94f0-6496e11942af.png">

### Reproduction Link
https://apexcharts.com/javascript-chart-demos/column-charts/basic/

**Comment by busyxiang** [comment]
@ryanklenk nope, still waiting for the dev team to fix this

**Comment by idpaterson** [comment]
I do not know how to account for the discrepancy between where Safari _thinks_ the legend elements are located and where they are rendered. In this example, Safari is indicating the bounding box of the black circle on the first legend item. 

<img width="376" alt="Screen Shot 2022-05-16 at 10 22 07 AM" src="https://user-images.githubusercontent.com/507058/168617018-86e38cee-cfb4-4599-9165-c10738bdc173.png">

We are currently testing `.apexcharts-svg .apexcharts-legend { position: fixed !important; }` as a quick and dirty workaround. The legend does not scale to width so it is not centered at different zoom levels, but it is at least at the correct position vertically.

**Comment by idpaterson** [comment]
I'm not seeing this glitch anymore. The CSS workaround is still in place in my implementation but it no longer has any effect because the legend has been moved outside of the SVG.  Currently using apexcharts 3.37.2, perhaps it was fixed by 088ed12 which was included in 3.37.1

**Comment by idpaterson** [comment]
Indeed, looks like 3.37.1 fixed it. Check out this codepen with Safari in [3.36.1](https://codepen.io/idpaterson/pen/oNaNpOz) where the legend bug still existed on page zoom in/out vs [3.37.2](https://codepen.io/idpaterson/pen/MWPWrxR) which seems to be fixed.

**Comment by brianlagunas** [comment]
Due to the feedback from the community, this appears to have been fixed and is no longer an issue. I will be closing it as fixed.

**Comment by VaibhavTessell** [comment]
I can see the issue fixed in Version 3.37.2, but seems like some regression happened in 3.41.0

Can somebody please look into it? @brianlagunas

**Comment by VaibhavTessell** [comment]
Can @busyxiang / @brianlagunas reopen this thread as I can see the issue getting reproducing in Version 3.41.0. Thank you!

**Comment by brianlagunas** [comment]
> @busyxiang / @brianlagunas Can you please reopen this thread as I can see the issue getting reproducing in Version 3.41.0 ? Thank you!

Do you have a codepen proving this?

**Comment by VaibhavTessell** [comment]
> Do you have a codepen proving this?

Link to the codepen with 3.41.0 version: https://codepen.io/Vaibhav-Singh-the-bold/pen/BaGJmjx

**Comment by mervekaraman** [comment]
Same error goes for zoom out in safari. Legend comes out of the div when zoom out

**Comment by DmitryKhiliuk** [comment]
Is there any way to solve this problem?

**Comment by NikitaM8** [comment]
Hello!
Still now solution? Using 3.53 version, legends are still not scale properly in Safari while zoom in/out :(

**Comment by junedchhipa** [comment]
Can someone mention the Safari version, operating system on which the bug can be reproduced? I am having a hard time reproducing it on Safari on mac

**Comment by NikitaM8** [comment]
Sure.
Facing with this issue on Safari Version 17.2.1 (19617.1.17.11.12)

**Comment by toddsiegel** [comment]
I am experiencing this as well with apexcharts 3.54.1, and Sarari Version 17.6 (19618.3.11.11.5)

**Comment by junedchhipa** [comment]
The issue is fixed in the latest version of apexcharts@4.5.0
